resources/server/sv_main.ts

```diff
@@ -65,17 +66,10 @@ async function getCredentials(identifier: string): Promise<string> {
   return number[0].phone_number;
 }
 
-on('playerConnecting', async () => {
-  const _source = getSource()
-  try {
-    const identifier = getIdentifier(_source);
-    await generatePhoneNumber(identifier)
-  } catch(e) {
-    mainLogger.error(`Failed to generate a phone number, ${e.message}`, {
-      source: _source,
-    })
-  }
-})
+onNet('esx:playerLoaded', async (playerId: number, xPlayer: any) => {
+  const identifier = xPlayer.identifier;
+  await generatePhoneNumber(identifier);
+});
```

Comment:
Might want to add a try catch with a `mainLogger.error` to provide a error log.

Reply:
We have a error log in `generatePhoneNumber` already so wouldn’t this be redundant? Just for phone number creation we have a catch there and on `generateRandomNumber` 🤔